utils/math/differential/Derivative.java

```diff
@@ -1,4 +1,4 @@
-package utils.math.differential;
+package frc.robot.common.utils.math.differential;
```

Comment:
I'm not sure, but I think you can WPILib's [`LinearFilter.backwardFiniteDifference`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/filter/LinearFilter.html#backwardFiniteDifference(int,int,double)) instead.